index.html

```diff
@@ -24,8 +24,11 @@
   <!-- bluebird only needed if this page needs to run on Internet Explorer -->
   <!-- msal.min.js can be used in the place of msal.js; included msal.js to make debug easy -->
   <script src="https://cdnjs.cloudflare.com/ajax/libs/bluebird/3.3.4/bluebird.min.js" class="pre"></script>
-  <script src="https://secure.aadcdn.microsoftonline-p.com/lib/1.1.2/js/msal.js"></script>
-  <script src="https://code.jquery.com/jquery-3.2.1.min.js" class="pre"></script>
+  <script type="text/javascript" src="https://alcdn.msauth.net/lib/1.2.1/js/msal.js" integrity="sha384-9TV1245fz+BaI+VvCjMYL0YDMElLBwNS84v3mY57pXNOt6xcUYch2QLImaTahcOP" crossorigin="anonymous"></script>
```

Comment:
>integrity= [](start = 85, length = 10)

what does this do?

Reply:
These are SRI hashes added for our CDN builds, more notes [here](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-core/README.md)

Reply:
cool. thanks.